### PR TITLE
metrics glossary: confusion matrix

### PR DESCRIPTION
**Pull request by y27choi**
### Linked issue(s):
Fixes KOL-2541

### What change does this PR introduce and why?
Addes confusion matrix metric doc

### Please check if the PR fulfills these requirements

- [ ] Include reference to internal ticket and/or GitHub issue "Fixes #NNNN" (if applicable)
- [ ] Relevant tests for the changes have been added
- [ ] Relevant docs have been added / updated

### CODE REVIEW

File: docs/metrics/confusion-matrix.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
+# Confusion Matrix
+
+!!! info inline end "Guide: True Positive / False Positive / False Negative / True Negative"
```
Comment (marko-polo-cheno) [inline docs/metrics/confusion-matrix.md:3]: Should this be conf mat?

File: docs/metrics/confusion-matrix.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+    You can find more info on true positive, false positive, false negative, true negative in [TP / FP / FN / TN](./tp-fp-fn-tn.md) guide.
```
Comment (marko-polo-cheno) [inline docs/metrics/confusion-matrix.md]: true positive, false positive, false negative,` or` true negative in` the` [] guide.

File: docs/metrics/confusion-matrix.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+
+A confusion matrix is a structured plot describing classification model performance as a table that highlights counts
+of objects with predicted classes (columns) against the actual classes (rows). Each cell has a count of the number of
+objects that have its correct class and predicted class, which indicates how confused a model is. A model is confused
+when a predicted class does not match the actual class. When they do match, this is considered a true positive (TP). In
+general, a model resulting in more TPs/true negatives (TNs) with fewer false positives (FPs)/false negatives (FNs) is
```
Comment (marko-polo-cheno) [inline docs/metrics/confusion-matrix.md]: add `true positives` for parallel structure

File: docs/metrics/confusion-matrix.md [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+
+|  | Predicted `Airplane` | Predicted `Boat` | Predicted `Car` |
+| --- | --- | --- | --- |
+| `Airplane` | 200 | 0 | 0 |
+| `Boat` | 100 | 8,800 | 600 |
+| `Car` | 100 | 0 | 1,000 |
+
+</center>
+
+This confusion matrix reveals that a model is very good at identifying `Boat`: 8,800 of 9,500 `Boat` were correctly
```
Comment (marko-polo-cheno) [inline docs/metrics/confusion-matrix.md]: For readability we could:
...identifying the `Boat` class: x of y `Boat`s were...

**Review by marko-polo-cheno — APPROVED** [review]

File: docs/metrics/confusion-matrix.md [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+### Normalization
+
+Sometimes it is easier to focus on **class-level behavior** if you are using a normalized confusion matrix. If confusion
+matrices are color-coded, normalizing can also create a better visual representation:
+
+![Example of normalized and colored confusion matrix](../assets/images/metrics-confusion-matrix-normalized-light.png#only-light)
+![Example of normalized and colored confusion matrix](../assets/images/metrics-confusion-matrix-normalized-dark.png#only-dark)
+
+You can normalize a confusion matrix by `row` (actual classes), `column` (predicted classes), or `all` (entire matrix).
+Each type of normalization surfaces a view sharing different information, which is outlined below.
```
Comment (munkyshi) [inline docs/metrics/confusion-matrix.md:145]: Is it worth linking normalization by row / column to recall / precision?

**Review by munkyshi — APPROVED** [review]
One small non-blocking question.

File: docs/metrics/confusion-matrix.md (hunk 5 above)
Comment (y27choi) [reply docs/metrics/confusion-matrix.md:145]: I like that. I added a line that describes the relationship between the row / column normalization to recall / precision.

**Pull request merged**